.github/workflows/codeql.yml

```diff
@@ -0,0 +1,126 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  # push:
+  #   branches: [ "main", "master" ]
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
Remove these two lines. I don't want this to run periodically. For context - we are trying to switch our CI actions to all be executed on the commit or push rather than wall clock time.